kobo/apps/accounts/tests/test_current_user.py

```diff
@@ -51,11 +51,12 @@ def test_update_extra_detail(self):
         # …and what we didn't touch should still be there as well
         assert response_extra_details['name'] == 'SpongeBob'
 
+    @override_config(
+        USER_METADATA_FIELDS=json.dumps(
```

Comment:
`json.dumps` is perfectly fine, but we use a helper `LazyJSONSerializable` for specific settings in `Constance` on production, I think it would better to be consistent in the test too. 
 
See https://github.com/kobotoolbox/kpi/blob/f043d71847d51ccfe5a057c63382e088e3d85cb9/kobo/apps/accounts/tests/test_forms.py#L52-L54